CMakeLists.txt

```diff
@@ -306,6 +306,15 @@ else()
     endif()
 endif()
 
+if(NOT WIN32 AND NOT APPLE)
+  find_library(LIBMLIRMIOPEN MLIRMIOpen)
```

Comment:
Please make it similarly to COMGR path. The use of MLIRMIOpen should be explicitly asked by cmake variable. If MLIRMIOpen not found, build should fail. This is important for testing.

Reply:
> The use of MLIRMIOpen should be explicitly asked by cmake variable.

Not resolved.

Reply:
Maybe I understand it wrong: I added the `REQUIRED` keyword in previous commit and thought that full-filled the requirements.

Reply:
Resolved. Added `MIOPEN_USE_MLIR` CMAke flag. MLIR will be used only if explicitly asked.